core/src/main/java/org/elasticsearch/index/query/GeoDistanceQueryBuilder.java

```diff
@@ -79,7 +80,9 @@
     @Deprecated
     private static final ParseField OPTIMIZE_BBOX_FIELD = new ParseField("optimize_bbox")
             .withAllDeprecated("no replacement: `optimize_bbox` is no longer supported due to recent improvements");
-    private static final ParseField DISTANCE_TYPE_FIELD = new ParseField("distance_type");
+    @Deprecated
+    private static final ParseField DISTANCE_TYPE_FIELD = new ParseField("distance_type")
+            .withAllDeprecated("no replacement: `distance_type` is no longer supported due to recent improvements");
```

Comment:
"due to recent improvements" doesn't give the user any information about what they should do instead. Can you make the message a little more descriptive in pointing to the improvements that obviated the need for this setting?